### PR TITLE
AM Data Location

### PR DESCRIPTION
**Pull request by juanpablotimpanaro**

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -53,14 +53,12 @@ See xref:logs.adoc[Logs].
 
 Limits on data storage and retention are determined by your subscription level (Base or Titanium). All customers receive Anypoint Monitoring at the Base subscription level.
 
-For Gold and Platinum subscriptions, data is stored in US East (N. Virginia).
+* For Gold/Platinium
+** Metric data in the same region as your control plane: US East (N. Virginia) or EU (Frankfurt)
```
Comment (jiang-wu) [inline modules/ROOT/pages/index.adoc:57]: Stores metric data ...

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -53,14 +53,12 @@ See xref:logs.adoc[Logs].
...
 
-The Titanium subscription:
-
-* Significantly increases the storage per vCore over the Base subscription.
-* Stores log and metric data in the same region as your control plane. These regions are available:
-** US East (N. Virginia)
-** EU (Frankfurt)
+* For Titanium
+** Significantly increases the storage per vCore over the Base subscription.
+** Log and metric data in the same region as your control plane: US East (N. Virginia) or EU (Frankfurt)
```
Comment (jiang-wu) [inline modules/ROOT/pages/index.adoc:61]: Stores log and metric ...

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -53,14 +53,12 @@ See xref:logs.adoc[Logs].
 
 Limits on data storage and retention are determined by your subscription level (Base or Titanium). All customers receive Anypoint Monitoring at the Base subscription level.
 
-For Gold and Platinum subscriptions, data is stored in US East (N. Virginia).
+* For Gold/Platinium
```
Comment (jiang-wu) [inline modules/ROOT/pages/index.adoc:56]: For Gold and Platinum subscriptions

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -53,14 +53,12 @@ See xref:logs.adoc[Logs].
...
+** Metric data in the same region as your control plane: US East (N. Virginia) or EU (Frankfurt)
 
-The Titanium subscription:
-
-* Significantly increases the storage per vCore over the Base subscription.
-* Stores log and metric data in the same region as your control plane. These regions are available:
-** US East (N. Virginia)
-** EU (Frankfurt)
+* For Titanium
```
Comment (jiang-wu) [inline modules/ROOT/pages/index.adoc:59]: For Titanium subscription

**Review by jiang-wu — CHANGES_REQUESTED** [review]

**Review by jiang-wu — APPROVED** [review]

**Comment by jiang-wu** [comment]
For https://www.mulesoft.org/jira/browse/DOCS-10085

**Review by michael-lee-mulesoft — APPROVED** [review]

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**